cmd/job.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/go-playground/validator/v10"
+	"github.com/odpf/guardian/app"
+	"github.com/odpf/guardian/internal/crypto"
+	"github.com/odpf/guardian/jobs"
+	"github.com/odpf/guardian/plugins/notifiers"
+	"github.com/odpf/salt/log"
+	"github.com/spf13/cobra"
+)
+
+func JobCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "job",
+		Aliases: []string{"jobs"},
+		Short:   "Manage jobs",
+		Example: heredoc.Doc(`
+			$ guardian job run fetch_resources
+		`),
+	}
+
+	cmd.AddCommand(
+		runJobCmd(),
+	)
+
+	cmd.PersistentFlags().StringP("config", "c", "./config.yaml", "Config file path")
+	cmd.MarkPersistentFlagFilename("config")
+
+	return cmd
+}
+
+func runJobCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "run",
+		Short: "Fire a specific job",
+		Example: heredoc.Doc(`
+			$ guardian job run fetch_resources
```

Comment:
When there is a typo with the job name it shows like this
```
$ guardian job run fetch_resources1111          
invalid argument "fetch_resources1111" for "guardian job run"
```
should we add more information e.g. suggestions/shows some supported jobs?

Reply:
couldn't find a way how to add custom message for arg validation in cobra. @mabdh do you have any suggestions?

Reply:
yes there is no simple solution with cobra for this, I think we can just keep it like this since this not really crucial to do.